target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -9,48 +9,29 @@ board=$(board_name)
 case "$FIRMWARE" in
 "ath9k-eeprom-ahb-18100000.wmac.bin")
 	case $board in
-	8dev,lima)
-		caldata_extract "art" 0x1000 0x800
-		;;
-	aruba,ap-115)
-		caldata_extract "oemdata" 0x1000 0x440
-		;;
-	asus,pl-ac56|\
-	asus,rp-ac51|\
-	asus,rp-ac66)
-		caldata_extract "art" 0x1000 0x440
-		;;
 	avm,fritz1750e|\
 	avm,fritz4020|\
 	avm,fritz450e|\
 	avm,fritzdvbc)
 		caldata_extract_reverse "urlader" 0x1541 0x440
 		;;
-	dlink,dir-505)
-		caldata_extract "art" 0x1000 0x440
-		ath9k_patch_mac $(mtd_get_mac_text "mac" 0x4)
-		;;
 	dlink,dir-842-c1|\
 	dlink,dir-842-c2|\
 	dlink,dir-842-c3|\
 	nec,wf1200cr|\
 	nec,wg1200cr|\
 	wd,mynet-n600|\
 	wd,mynet-n750)
-		caldata_extract "art" 0x1000 0x440
 		ath9k_patch_mac $(mtd_get_mac_ascii devdata "wlan24mac")
 		;;
```

Comment:
Shoudn't ```ath9k_patch_mac``` also be removed and converted to **10_fix_wifi_mac**?
```ath9k_patch_mac``` applies obtained MAC address to calibration data extracted by ```caldata_extract```.

Reply:
Maybe in the case of devices where we need to change the MAC address read as an ASCII variable, it is better to leave the current solution?
It is possible to read the MAC address as ASCII from dts, but only if it is located at a fixed address in memory or as the `ethaddr` variable from the u-boot env partition.

Reply:
This is now done.